cclib/method/ddec.py

```diff
@@ -271,27 +272,94 @@ def calculate(self, indices=None, fupdate=0.05):
             self.radial_grid_r.append(r)
 
         # Carry out step 3 of DDEC6 algorithm [Determine conditioned charge density and tau]
-        self.logger.info("Conditioning charge densities.")
+        self.logger.info("Conditioning charge densities. (Step 3/7)")
         self.condition_densities()
 
+        # STEP 4-7
+        # Optimize weights on each grid point
+        self._kappa = [0.0 for x in self.data.atomnos]
+        update_kappa = False
+        self.N_A = []
```

Comment:
Short name. Could you either comment what this is or use a more verbose name?

Reply:
These were from the paper. I've added equation number reference for both of them. Thanks.